src/test/e2e/21_connect_creds_test.go

```diff
@@ -68,7 +71,7 @@ func TestMetrics(t *testing.T) {
 
 	tunnel, err := c.NewTunnel("zarf", "svc", "agent-hook", "", 8888, 8443)
 	require.NoError(t, err)
-	_, err = tunnel.Connect()
+	_, err = tunnel.Connect(context.TODO())
```

Comment:
Same TODO question as above (I know these ones are tests and the others weren't, but maybe still valid)

Reply:
fixed